### DISCUSSION TITLE
Release of version 4.12

### DISCUSSION THREAD

**Discussion by JCHacking** [DependencyTrack/dependency-track#3949] [General]
Sorry to bother, but I recently developed a new feature (#1674) and I need it as soon as possible.

Is there any tentative release date for version 4.12? How can I help to get it released as soon as possible?

Thanks!

**Comment by nscuro** [comment]
There's no fixed date, you can see the remaining issues in the v4.12 milestone here: https://github.com/DependencyTrack/dependency-track/milestone/27
